UndertaleModTool/Scripts/Builtin Scripts/ShowRoomName.csx

```diff
@@ -1,30 +1,50 @@
+
 // Adds room ID and name display under the debug mode timer display
 
 EnsureDataLoaded();
-
+ScriptMessage("Show room name and ID in debug mode\nby krzys_h, Kneesnap\n\n\nDeltarune Ch2 Patch By GattoDev");
+Data.Functions.EnsureDefined("room_get_name", Data.Strings); // required for Deltarune
```

Comment:
If this is required for Deltarune, then shouldn't you move that line below, to the block that checks for Deltarune?

Reply:
@EvaxTheFox is this only required for deltarune, or also required for undertale?
If the former, then it would be better if it'd be moved, if the latter, then don't mind me.